src/components/Articles.jsx

```diff
@@ -11,14 +13,15 @@ const Articles = () => {
       .then((result) => {
         setArticles(result);
         setIsLoading(false);
+        console.log(result);
       })
       .catch((err) => {
         setError(true);
       });
-  }, []);
+  });
```

Comment:
You have an infinite loop appearing in your code! :( 

With your `useEffect` you'll need to include a dependency array as at the moment, when the page is rendered it will call getArticles and set state which will cause a rerender. But because we don't have a dependency array it is getting in a loop of constantly setting state and rerendering.

`useEffect(() => {}, [])`